benchmarks/run_on_openml_datasets.py

```diff
@@ -163,21 +169,22 @@
                         )
                         low_scores[task_id] = np.mean(scores)
             else:
-                logger.warning(f"No OpenML scores available for {task_id}")
+                logger.debug(f"No OpenML scores available for {task_id}")
         except Exception as e:
             logger.warning(
                 constraint_error_template.format(
-                    id=task_id, reason=f"of an unhandled error: {e}"
+                    id=task_id, reason=f"of an unhandled error: {e}."
                 )
             )
             errors[task_id] = e
+            errors_one_line[task_id] = str(e).split("\n")[-2]
             continue
 
 logger.info(f"Finished! ")
 logger.error(f"{len(errors)} tasks with errors: {set(errors.keys())}")
-# show all unique errors
-for error in set(errors.values()):
-    logger.error(f"Error: {error}")
+# print all unique errors
+for error in set(errors_one_line.values()):
+    logger.error(f"Error: {error}, {list(errors_one_line.values()).count(error)} times")
```

Comment:
Maybe this would be cleaner? And more efficient (although it's probably kind of irrelevant for this case)
```suggestion
errors_counter = Counter(errors_one_line.values())
for error, count in errors_counter.items():
    logger.error(f"Error: {error}, {count} times")
```
`Counter` needs to be imported from `collections`.